src/Behat/Behat/Tester/Runtime/RuntimeStepTester.php

```diff
@@ -122,12 +137,48 @@ private function testDefinition(Environment $env, FeatureNode $feature, StepNode
         }
 
         $call = $this->createDefinitionCall($env, $feature, $search, $step);
-        $result = $this->callCenter->makeCall($call);
+
+        $lambda = function() use ($call) {
+          return $this->callCenter->makeCall($call);
+        };
+
+        // @todo We can only "spin" if we are interacting with a remote browser. If the browser is
+        // running in the same thread as this test (such as with Goutte or Zombie), then spinning
+        // will only prevent that process from continuing, and the test will either pass immediately,
+        // or not at all. We need to find out how to check what Driver we're using...
```

Comment:
anything talking about browser should be removed. Behat does not know about browsers